src/components/sidebar/types.d.ts

```diff
@@ -0,0 +1,15 @@
+type NodeT = import("react").ReactNode
+
+interface SidebarProps {
+  info?: NodeT
+  right?: boolean
+  children?: NodeT
+}
+
+interface PortalSidebarProps<T = any> {
+  isOpen?: boolean
+  closeOnEsc?: boolean
+  onClose?: (args?: T) => void
+  right?: boolean
+  children?: NodeT
```

Comment:
Would a `disabled` optional property make sense here so we can disable it?

Reply:
is `isOpen` setted to false is eq to your proposal?

Reply:
`isOpen` removed. As far as all controls will be in the parent component.